### PR TITLE
French translation of tour-of-scala.md

### PR DESCRIPTION
**Pull request by agnesmdev**

### CODE REVIEW

**Comment by SethTisue** [comment]
any French speakers out there who could review this?

**Comment by SethTisue** [comment]
@leobenkel I tried merging https://github.com/scala/docs.scala-lang/pull/1799 here, but it didn't merge cleanly. I think the easiest thing here might be if I merged this PR first, and then you can submit your changes in a followup PR. how's that sound?

**Comment by SethTisue** [comment]
oh never mind, I was confused. let's close this one and proceed on #1799

File: _fr/tour/tour-of-scala.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+façon concise, élégante et robuste. Il intègre sans problème les fonctionnalités des langages orientés objet et des
+langages fonctionnels.
+
+## Scala est orienté objet ##
+Scala est un langage purement orienté objet dans le sens où [toute valeur est un objet](unified-types.html).
+Les types et les comportements de ces objets sont décrits par des [classes](classes.html) et des trait [traits](traits.html).
+Les classes peuvent être étendues à travers des sous-classes et grâce à un système flexible de[composition de classes](mixin-class-composition.html).
+
+## Scala est fonctionnel ##
+Scala est également un langage fonctionnel dans le sen où [toute fonction est une valeur](unified-types.html).
```
Comment (phderome) [inline _fr/tour/tour-of-scala.md:31]: replace "le sen" with "le sens"

Comment (SethTisue) [reply _fr/tour/tour-of-scala.md:31]: PR welcome

**Pull request closed without merging**